packages/embark-pipeline/src/index.js

```diff
@@ -237,7 +236,8 @@ class Pipeline {
           action: constants.pipeline.init,
           options: {
             webpackConfigName: self.webpackConfigName,
-            pipelineConfig: self.pipelineConfig
+            pipelineConfig: self.pipelineConfig,
+            fs: self.embark.fs
```

Comment:
This can go away

Reply:
It's not being used as we've introduced separate fs utilities due to not being able to send function structures through process.send()